misc/config_tools/data/tgl-vecow-spc-7100-Corei7/partitioned.xml

```diff
@@ -0,0 +1,181 @@
+      <MAX_IOAPIC_LINES>120</MAX_IOAPIC_LINES>
+      <MAX_PT_IRQ_ENTRIES>64</MAX_PT_IRQ_ENTRIES>
+      <MAX_MSIX_TABLE_NUM>64</MAX_MSIX_TABLE_NUM>
+      <MAX_EMULATED_MMIO>16</MAX_EMULATED_MMIO>
+    </CAPACITIES>
+    <MISC_CFG>
+      <GPU_SBDF>0x00000010</GPU_SBDF>
+    </MISC_CFG>
+  </hv>
+  <vm id="0">
+    <vm_type>PRE_STD_VM</vm_type>
+    <name>PRE_STD_VM0</name>
+    <guest_flags>
+      <guest_flag>0</guest_flag>
+    </guest_flags>
+    <cpu_affinity>
+      <pcpu_id>0</pcpu_id>
+      <pcpu_id>2</pcpu_id>
+    </cpu_affinity>
+    <clos>
+      <vcpu_clos>0</vcpu_clos>
+      <vcpu_clos>0</vcpu_clos>
+    </clos>
+    <epc_section>
+      <base>0</base>
+      <size>0</size>
+    </epc_section>
+    <memory>
+      <start_hpa>0x100000000</start_hpa>
+      <size>0x20000000</size>
+      <start_hpa2>0x0</start_hpa2>
+      <size_hpa2>0x0</size_hpa2>
+    </memory>
+    <os_config>
+      <name>YOCTO</name>
+      <kern_type>KERNEL_BZIMAGE</kern_type>
+      <kern_mod>Linux_bzImage</kern_mod>
+      <ramdisk_mod>Ubuntu</ramdisk_mod>
+      <bootargs> rw rootwait root=/dev/sda3 console=ttyS0 noxsave nohpet no_timer_check ignore_loglevel log_buf_len=16M consoleblank=0 tsc=reliable reboot=acpi
+        </bootargs>
+    </os_config>
+    <legacy_vuart id="0">
+      <type>VUART_LEGACY_PIO</type>
+      <base>COM1_BASE</base>
+      <irq>COM1_IRQ</irq>
+    </legacy_vuart>
+    <legacy_vuart id="1">
+      <type>VUART_LEGACY_PIO</type>
+      <base>COM2_BASE</base>
+      <irq>COM2_IRQ</irq>
+      <target_vm_id>1</target_vm_id>
+      <target_uart_id>1</target_uart_id>
+    </legacy_vuart>
+    <console_vuart id="0">
+      <base>INVALID_PCI_BASE</base>
+    </console_vuart>
+    <communication_vuart id="1">
+      <base>INVALID_PCI_BASE</base>
+      <target_vm_id>1</target_vm_id>
+      <target_uart_id>1</target_uart_id>
+    </communication_vuart>
+    <pci_devs>
+      <pci_dev>00:17.0 SATA controller: Intel Corporation Device a0d3 (rev 20)</pci_dev>
+      <pci_dev>00:1f.6 Ethernet controller: Intel Corporation Device 15fb (rev 20)</pci_dev>
+      <pci_dev/>
+    </pci_devs>
+    <mmio_resources>
+      <TPM2>n</TPM2>
+    </mmio_resources>
+  </vm>
+  <vm id="1">
+    <vm_type>PRE_STD_VM</vm_type>
+    <name>PRE_STD_VM1</name>
+    <guest_flags>
+      <guest_flag>0</guest_flag>
+    </guest_flags>
+    <cpu_affinity>
+      <pcpu_id>1</pcpu_id>
+      <pcpu_id>3</pcpu_id>
+    </cpu_affinity>
+    <clos>
+      <vcpu_clos>0</vcpu_clos>
+      <vcpu_clos>0</vcpu_clos>
+    </clos>
+    <epc_section>
+      <base>0</base>
+      <size>0</size>
+    </epc_section>
+    <memory>
+      <start_hpa>0x120000000</start_hpa>
+      <size>0x20000000</size>
+      <start_hpa2>0x0</start_hpa2>
+      <size_hpa2>0x0</size_hpa2>
+    </memory>
+    <os_config>
+      <name>YOCTO</name>
+      <kern_type>KERNEL_BZIMAGE</kern_type>
+      <kern_mod>Linux_bzImage</kern_mod>
+      <ramdisk_mod>Ubuntu</ramdisk_mod>
```

Comment:
This is odd, `Ubuntu` is the tag name for a Yocto-bsed VM?

Reply:
@jackwhich, please help to update.

Reply:
FIX